app/src/main/java/org/mozilla/fenix/settings/search/SearchEngineListPreference.kt

```diff
@@ -76,6 +76,11 @@ abstract class SearchEngineListPreference @JvmOverloads constructor(
             it.identifier == defaultEngine
         } ?: searchEngineList.list.first()).identifier
 
+        context.components.search.searchEngineManager.defaultSearchEngine =
```

Comment:
This would work but I don't think it's the right place to set the default.
This adds another responsibility to a method that should just "refreshSearchEngineViews".
If the issue is in `deleteSearchEngine()` maybe we can fix it there?